internal/proxy/util.go

```diff
@@ -22,16 +22,19 @@ import (
 	"strconv"
 	"strings"
 
+	"go.uber.org/zap"
+
+	"github.com/milvus-io/milvus/internal/log"
 	"github.com/milvus-io/milvus/internal/proto/commonpb"
 	"github.com/milvus-io/milvus/internal/proto/schemapb"
-	"github.com/milvus-io/milvus/internal/util/funcutil"
 )
 
 // enableMultipleVectorFields indicates whether to enable multiple vector fields.
 const enableMultipleVectorFields = false
 
 // maximum length of variable-length strings
-const maxVarCharLength = int64(65535)
+const maxVarCharLengthKey = "max_length_per_row"
```

Comment:
is it max_length_per_row or max_length_per_field?

Reply:
> is it max_length_per_row or max_length_per_field?

Will max_length_per_field be misinterpreted as the maximum number of rows per field?